pkg/storage/store_test.go

```diff
@@ -229,10 +228,3 @@ func verifyItr(t *testing.T, itr storage.StoreIterator, count int, prefix string
 	require.Empty(t, itr.Value())
 	require.Error(t, itr.Error())
 }
-
```

Comment:
failure while creating table db_prefix_key631219b5-a59f-44b3-94f0-52b52e21c6a6: Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-a59f-44b3-94f0-52b52e21c6a6.db_prefix_key631219b5-a59f-44b3-94f0-52b52e21c6a6 . This uuid with new change of database.tableName it keeps failing with sql instance.

Reply:
Quoting the table name with the backtick (`) should do the trick.

[Permitted characters in MySQL object identifiers](https://dev.mysql.com/doc/refman/8.0/en/identifiers.html):

> * Permitted characters in unquoted identifiers:
>    * ASCII: [0-9,a-z,A-Z$_] (basic Latin letters, digits 0-9, dollar, underscore)
>    * Extended: U+0080 .. U+FFFF 
> 
> * Permitted characters in quoted identifiers include the full Unicode Basic Multilingual Plane (BMP), except U+0000:
>   * ASCII: U+0001 .. U+007F
>   * Extended: U+0080 .. U+FFFF 
> * ASCII NUL (U+0000) and supplementary characters (U+10000 and higher) are not permitted in quoted or unquoted identifiers.
> * Identifiers may begin with a digit but unless quoted may not consist solely of digits.
> * Database, table, and column names cannot end with space characters. 
> 
> The identifier quote character is the backtick (`):

Reply:
Please leave the randomized table names in the tests... it allows fast test re-runs without incurring the risk of test failures due to naming conflicts.

Reply:
updated